feat(content): New Outer Rim culture conversations and news entries (#10409)
endless-sky/endless-sky: data/human/culture conversations.txt

			label "end"
			`	Abernathy doesn't slow down until he's within an arm's length of the finishing line; as he crosses it he seizes the tape, making a yelp of simultaneous joy and relief. He holds the tape up before everybody in the crowd and at the cameras pointed at him, taking in the attention (and catching his breath) before accepting a bottle of water from someone on the sidelines.`
				decline

mission "CCOR Guerrilla Standoff"
	minor
	source "Gagarin"
	to offer
		random < 1
	on offer
		conversation

> 			`Every map of human space places Gagarin in the list of seedy, go-at-your-own-risk pirate ports which would bring to mind the likes of Greenrock's fighting pits and Freedom's gear bazaars. Still, it appears much like any other old, slightly run-down Southern spaceport, only at a much grander scale. Being the center of the planet's extensive transportation system, it seems as if you are wading through thousands of Gagarians and offworlders in every atrium and hallway of this spaceport, as they do the same to make their connections on- and off-world.`

Reviewer: What's a gear bazaar?

Reviewer 2: I ended up going into quite a rabbit hole with this. Short answer is that I read it as an higher-tech black market for more unusual outfits, longer answer that may or may not be useful below.

The way I read it was that a gear bazaar would be a higher-tech black market where you might buy illegal or unusual outfits/items. Looking at planet descriptions now though, Freedom seems to me more agricultural based than likely to be renowned for a seedy black market. 

Some other planets that might fit a famous gear bazaar better:
- **Haven** - another North Pirate planet, description notes that it sells illegal outfits and modified ships
- **Covert** - Core, the mining cartel seems to be running one of the more dangerous markets, though the focus is more on nuclear materials and slave-labor goods. Alternative name: _Radium Exchange_
- **Bloodsea** - South, their only industry is selling stolen cargo, so it’d make sense that a gear bazaar is the main thing of note.

Planet & spaceport descriptions of each:

**Freedom**
description `Freedom is home to the oldest and largest anarchist colonies in human history. The original settlers arrived in the first century after the invention of the hyperdrive, coming with the goal of simply getting as far away from civilization as possible. The colony has reinvented itself countless times since then as different leaders have taken control, and in fact at present it is split into two separate towns, East Mesa and West Mesa.
	The environment is a mix of deserts and brackish oceans, with a fair amount of native wildlife and good potential for farming. Anywhere else in human space, this planet would have become densely settled, but out here it has remained mostly empty.`
	
spaceport `Apparently anarchists are not opposed to commerce as long as no one is regulating it. Half of the landing pads in the spaceport are available for ships; the other half have been taken over by farmers busy selling their produce, or merchants buying from incoming ships and piling crates into large trucks.
	A sign above the entrance to the spaceport bar reads, "Fine home-brewed whiskey! Get smashed without getting taxed!" Further down the street, a sign over what appears to be a real estate office reads, "Why not settle where The Man can't reach you?"
	Inexplicably, the only advertising on the seafood restaurant across the street is an enormous banner proclaiming, "We have brown rice."`
	
**Haven**
description `Haven was founded by highly successful "privateers" nearly two centuries ago. Since then, as rival bands of pirates have fought over it, it has changed hands fourteen times, but has never been successfully invaded by the Republic. Numerous small manufacturing shops sell outfits that cannot be bought legally, and there is even a small shipyard where customized ships are sold to those who have the right connections.
	Much of the planet is desert, but with enough moisture to grow enough crops to feed the entire population. Strip mining has turned other parts of the planet into barren wastelands, and has polluted many of the rivers.`
	
spaceport `The spaceport is old and dingy, with piles of litter in the gutters and graffiti on many of the alley walls. Many of the shops are closed, the windows boarded over or covered with yellowing paper.
	This is an independent world, and a relatively stable settlement, but the effects of its separation from the rest of the galaxy are evident. A few young people carrying large backpacks eye you with interest, perhaps trying to gauge their chances of getting you to give them a cheap lift off-planet.`

**Covert**
description `Covert is home to a powerful mining cartel, operating outside the control of Republic law or any other external authority. It is rumored to be one of the few planets in human space where weapons-grade uranium and plutonium is mined and enriched. Any goods for sale here were most likely produced by slave labor, often by survivors of ships that have been captured by pirates. A massive pirate fleet is docked on the outskirts of the spaceport, a warning to any private pilot who might think of cheating or opposing the cartel.`

spaceport `From the moment you landed, you have been followed by a pair of security guards with earpieces and laser guns. The spaceport is one enormous building, with square, straight hallways and featureless walls, and nearly every door you pass is locked. Cameras at each hallway intersection pivot to watch as you approach. This is simultaneously one of the most secure spaceports you have visited, and one of the most dangerous, although if you do not do anything to provoke the animosity of the cartel, you will probably be able to leave in peace.`
	
**Bloodsea**
description `Bloodsea is nearly uninhabited, except for a few pirate outposts. Its name comes from the planet's oceans, which are blood-red due to rhodophyte algae. Although the climate is temperate and there are many white sand beaches, no one comes here hoping to go swimming.
	The pirates have no industry of their own, but instead make a living by reselling stolen cargo from raids on merchant fleets.`
	spaceport `When you landed your ship near one of the larger outposts, no one even came out to greet you or to check your paperwork - not that anyone on a pirate world would care about such things. In the center of town is an open-air marketplace where large men armed with large guns stand guard over piles of crates, while others examine the wares that are for sale and haggle over prices.
	The local slang incorporates threats of extreme violence as a seemingly meaningless verbal tic. "Hey, I'm gonna cut off your fingers one by one until you buy a meat patty!" a vendor calls out cheerfully as you pass. "First one's free, and I'll tie a knot in your spine if you don't like it."`

Author: Very interesting… but the way I meant it, “gear” is slang for drug in NZ, so it would be some sort of drug emporium. I am completely happy with the above interpretation though.

Reviewer 2: Ah, that'll do it. In that case I think Freedom actually fits quite well as a drug emporium.

> 			`Although every map of human space places Gagarin in the list of seedy, go-at-your-own-risk pirate ports which would bring to mind the likes of Greenrock's fighting pits and Freedom's gear bazaars, by comparison Gagarin appears much more like any old, slightly run-down port a captain would find around the South, only at a much grander scale. Being the center of the planet's extensive transportation system, it seems as if you are wading through thousands of Gagarians and offworlders in every atrium and hallway of this spaceport, as they do the same to make their connections on- and off-world.`

Reviewer 3: Oh, boy, this first sentence is long. Maybe:
```suggestion
			`Every map of human space places Gagarin in the list of seedy, go-at-your-own-risk pirate ports which would bring to mind the likes of Greenrock's fighting pits and Freedom's gear bazaars. Still, it appears much like any other old, slightly run-down Southern spaceport, only at a much grander scale. Being the center of the planet's extensive transportation system, it seems as if you are wading through thousands of Gagarians and offworlders in every atrium and hallway of this spaceport, as they do the same to make their connections on- and off-world.`
```

			`	But as you make your way into the central hall you notice that, where you'd expect the greatest activity, everything has stopped. Even the ever-roving antigrav trolleys of cargo and supplies are backed up and motionless, and everybody's attention is directed at some sort of commotion in the very center of the spaceport.`
			choice
				`	(Take a look.)`
				`	(Go back to your ship.)`
					decline

> 			`	Stepping through the gaps within the established crowd you end up next to a spaceport gendarme, who appears as if he's meant to be on patrol duty but is too busy looking at what everybody else is looking at to do his job at the moment. Looking over the banisters beside him, you see first the enormous and oddly empty space in the middle of the hall, then a team of overalls-wearing men armed with drills and brushes. Surrounding them is a cordon of guerrillas in their rosy CCOR uniforms, sizing up the public and the spaceport security around them with their rifles and pistols.`

Author: ```suggestion
			`	Stepping through the gaps within the established crowd you end up next to a spaceport gendarme in their white uniform, who appears as if he's meant to be on patrol duty but is too busy looking at what everybody else is looking at to do his job at the moment. Looking over the banisters beside him, you see first the enormous and oddly empty space in the middle of the hall, then a team of overalls-wearing men armed with drills and brushes. Surrounding them is a cordon of guerrillas in their rosy CCOR uniforms, sizing up the public and the spaceport security around them with their rifles and pistols.`
```

> 			`	Stepping through the gaps within the established crowd you end up next to a spaceport gendarme, who appears as if he's meant to be on patrol duty but is too busy looking at what everybody else is looking at to do his job at the moment. You look over the bannisters with him and see, at first, the enormous, odd empty space in the middle of the hall; then a team of overalls-wearing men armed with drills and brushes. Surrounding them is a cordon of guerillas in their rosy CCOR uniforms, sizing up the public and the spaceport security around them with their rifles and pistols.`

Reviewer 3: Not sure the semicolon really works here, and that sentence seems a bit choppy. Maybe:
```suggestion
			`	Stepping through the gaps within the established crowd you end up next to a spaceport gendarme, who appears as if he's meant to be on patrol duty but is too busy looking at what everybody else is looking at to do his job at the moment. Looking over the bannisters beside him, you see first the enormous and oddly empty space in the middle of the hall, then a team of overalls-wearing men armed with drills and brushes. Surrounding them is a cordon of guerillas in their rosy CCOR uniforms, sizing up the public and the spaceport security around them with their rifles and pistols.`
```

Also, I think "bannister" is a British English spelling, and "guerilla" is an uncommon spelling, so maybe:
```suggestion
			`	Stepping through the gaps within the established crowd you end up next to a spaceport gendarme, who appears as if he's meant to be on patrol duty but is too busy looking at what everybody else is looking at to do his job at the moment. Looking over the banisters beside him, you see first the enormous and oddly empty space in the middle of the hall, then a team of overalls-wearing men armed with drills and brushes. Surrounding them is a cordon of guerrillas in their rosy CCOR uniforms, sizing up the public and the spaceport security around them with their rifles and pistols.`
```

> 			`	The painting guerrillas are busily finishing off the touches of what appears to be a statue of the burgundy-and-white emblem of the Confederated Councils, while a pile of rubble beside it suggests that a symbol of the local government once stood in its place. As they step away from the statue, a lone man in a much better-kept uniform than the others steps forward in front of their work. You can feel the air chilling as he raises his head to speak.`

Reviewer 4: I don't think these hyphens should be here.
```suggestion
			`	The painting guerrillas are busily finishing off the touches of what appears to be a statue of the burgundy and white emblem of the Confederated Councils, while a pile of rubble beside it suggests that a symbol of the local government once stood in its place. As they step away from the statue, a lone man in a much better-kept uniform than the others steps forward in front of their work. You can feel the air chilling as he raises his head to speak.`
```

			choice
				`	(Listen in.)`
				`	(Step away before he makes his speech.)`
					decline

> 			`	You don't see any amplification equipment, yet his voice fills up the room without effort.`

Author: Now that you mention it, probably only "fill".
```suggestion
			`	You don't see any amplification equipment, yet his voice fills the room without effort.`
```

			`	"Gagarians and offworlders. Captains and travellers. Servicemen and farmers.`
			`	"The worlds of the Outer Rim, and indeed, those of the Republic and the entirety of humanity, have and continue to be despoiled and devastated by ruthless profiteering.`

> 			`	"The executives of the Syndicate and the landed gentry of the Paradise Worlds steal the efforts of tens of billions of men, women, and children, from Prime to Foundry; from Valhalla to Earth; from Bourne to Hestia, and beyond. And for near a century they have colluded to impose their designs on our worlds! The corporate insurrection in Novo Minho and the gang wars in the northern Outer Rim which are only an obvious extension of the arm of people like Alastair Korban, who threaten to plunge our worlds into further suffering and bitter destruction!`

Author: ```suggestion
			`	"The executives of the Syndicate and the landed gentry of the Paradise Worlds steal the efforts of tens of billions of men, women, and children, from Prime to Foundry; from Valhalla to Earth; from Bourne to Hestia, and beyond. And for near a century they have colluded to impose their designs on our worlds! The corporate insurrection in Novo Minho and the gang wars in the northern Outer Rim are an obvious extension of the arm of people like Alastair Korban, only leading to further suffering and bitter destruction for our region!`
```

> 			`	"The executives of the Syndicate and the landed gentry of the Paradise Worlds steal the efforts of tens of billions of men, woman, and children, from Prime to Foundry; from Valhalla to Earth; from Bourne to Hestia, and beyond. And for near a century they have colluded to impose their designs on our worlds! The corporate insurrection in Novo Minho and the gang wars in the northern Outer Rim which are only an obvious extension of the arm of people like Alastair Korban, who threaten to plunge our worlds into further suffering and bitter destruction!`

Reviewer 3: ```suggestion
			`	"The executives of the Syndicate and the landed gentry of the Paradise Worlds steal the efforts of tens of billions of men, women, and children, from Prime to Foundry; from Valhalla to Earth; from Bourne to Hestia, and beyond. And for near a century they have colluded to impose their designs on our worlds! The corporate insurrection in Novo Minho and the gang wars in the northern Outer Rim which are only an obvious extension of the arm of people like Alastair Korban, who threaten to plunge our worlds into further suffering and bitter destruction!`
```

			`	"But there is still hope. We can bring about the independence of Gagarin and unify this world that has been lost to the Syndicate-sponsored oligarchs and their cronies and puppet officials. We can reunite the Outer Rim and restore the prosperity stolen from us. We can form a new Confederated Councils and once again shine the light of Belenos across the entire galaxy!"`

> 			`	There's a beat as the man finishes his speech to scattered applause, as the gendarmes look around and consider the response of the crowd around them. The speaker steps down and orders his guerrillas into formation before marching them out of the still crowd and into the back of a dusty, burgundy army van you hadn't noticed before, leaving the remains of the old statue scattered around the empty space.`

Reviewer 4: Using "beat" here doesn't quite sound right.
It might be because it says the beat occurs as the guy finishes his speech and people applaud. It seems kind of weird to sy that given that a "beat" is supposed to be some sort of pause. The second half of the sentence, about the gendarmes, also kind of seems like it should be its own sentence.

Author: The beat/pause and the gendarmes wondering what to do occur at the same time, so I chose to put them together in the same sentence.

Maybe better?
```suggestion
			`	There's scattered applause as the man finishes his speech to the spaceport, as the gendarmes look around and consider the response of the crowd around them. The speaker steps down and orders his guerrillas into formation before marching them out of the still crowd and into the back of a dusty, burgundy army van you hadn't noticed before, leaving the remains of the old statue scattered around the empty space.`
```

Reviewer 4: It should probably be "There *are* scattered applause...", "to the spaceport" seems unnecessary, and beginning the last clause with "as the" doesn't make sense to me. That part should probably be a new sentence:
```suggestion
			`	There are scattered applause as the man finishes his speech. The gendarmes look around and consider the response of the crowd around them. The speaker steps down and orders his guerrillas into formation before marching them out of the still crowd and into the back of a dusty, burgundy army van you hadn't noticed before, leaving the remains of the old statue scattered around the empty space.`
```
Or, perhaps:
```suggestion
			`	The gendarmes look around and consider the response of the crowd around them as the man finishes his speech to scattered applause. The speaker steps down and orders his guerrillas into formation before marching them out of the still crowd and into the back of a dusty, burgundy army van you hadn't noticed before, leaving the remains of the old statue scattered around the empty space.`
```

Reviewer 3: Probably "is scattered applause"? Unless people clap, then stop, then clap again, and so on.

> 			`	There's a beat as the man finishes his speech to scattered applause, as the white-uniformed gendarmes look around and consider the response of the crowd around them. The speaker steps down and orders his guerrillas into formation before marching them out of the still crowd and into the back of a dusty, burgundy-colored army van you hadn't noticed before, leaving the remains of the old statue scattered around the empty space.`

Reviewer 4: ```suggestion
			`	There's a beat as the man finishes his speech to scattered applause, as the white-uniformed gendarmes look around and consider the response of the crowd around them. The speaker steps down and orders his guerrillas into formation before marching them out of the still crowd and into the back of a dusty, burgundy army van you hadn't noticed before, leaving the remains of the old statue scattered around the empty space.`
```

Reviewer 4: I'm also not sure the gendarmes should be described as "white-uniformed" here. If that's an important piece of description, it ought to be provided earlier, when we find ourselves beside one.

Author: ```suggestion
			`	There's a beat as the man finishes his speech to scattered applause, as the gendarmes look around and consider the response of the crowd around them. The speaker steps down and orders his guerrillas into formation before marching them out of the still crowd and into the back of a dusty, burgundy army van you hadn't noticed before, leaving the remains of the old statue scattered around the empty space.`
```

> 			`	There's a beat as the man finishes his speech to scattered applause, as the white-uniformed gendarmes look around and consider the response of the crowd around them. The speaker steps down and orders his guerrillas to formation before marching them out of the still crowd and into the back of a dusty, burgundy-colored army van you hadn't noticed before, leaving the remains of the old statue scattered around the empty space.`

Reviewer 3: "to formation" makes it sound like formationing is a verb. It sort of works in this context, but I think:
```suggestion
			`	There's a beat as the man finishes his speech to scattered applause, as the white-uniformed gendarmes look around and consider the response of the crowd around them. The speaker steps down and orders his guerrillas into formation before marching them out of the still crowd and into the back of a dusty, burgundy-colored army van you hadn't noticed before, leaving the remains of the old statue scattered around the empty space.`
```
would work better.

			`	You turn around and see the gendarme you ended up next to. He meets your eyes and recognizes your off-world air.`
			`	"Just an everyday interruption, Captain," he says. "Only nostalgists for the old cause. We get a dozen of these incidents a month." He seems to be interrupted by a buzz from the radio receiver at his hip and strides off into the crowd, leaving you on the ledge.`

> 			`	Slowly but surely, the spaceport snaps out of its shock and picks up its previous momentum, although as you turn around you pass two government-branded trucks mounted with cranes, almost certainly there to carry off the newly-installed statue and to clean up the mess left behind by the guerrillas.`

Reviewer 5: ```suggestion
			`	Slowly but surely, the spaceport snaps out of its shock and picks up its previous momentum, although as you turn around you pass two government-branded trucks mounted with cranes, almost certainly there to carry off the newly installed statue and to clean up the mess left behind by the guerrillas.`
```

Author: I forgot how to use dashes lol

> 			`	Slowly but surely, the spaceport snaps out of its shock and picks up its previous momentum, although as you turn around you pass two government-branded trucks mounted with cranes, almost certainly there to carry off the statue that was just installed and to clean up the mess left behind by the guerrillas.`

Reviewer 3: Last sentence is a bit wordy. Maybe:
```suggestion
			`	Slowly but surely, the spaceport snaps out of its shock and picks up its previous momentum, although as you turn around you pass two government-branded trucks mounted with cranes, almost certainly there to carry off the newly-installed statue and to clean up the mess left behind by the guerrillas.`
```

mission "Raider returning to Mordente-Bridi"
	source "Mordente-Bridi"
	to offer

> 		random < 1

Reviewer 3: These seem almost too rare for me, although I guess they match other culture conversations. Might be something to look at rebalancing in the future, or something to patch by adding another few dozen culture conversations.

	on offer
		conversation
			`Watching the microgravity section of the spaceport from one of the bars in Mordente Cylinder, you observe a Bastion in pirate markings coming nearly too fast for safe approach to the station, its hull covered in gashes and burns from what looks to be Plasma Cannon shots.`

> 			`	As it comes to a stop in a maintenance bay, the spaceport's engineering shuttles appear from all over and begin buzzing around the Bastion, assessing the damage and beginning the work on repairs; meanwhile, its crew disperse from the ship and immediately begin unloading a multitude of busted-up crates and containers from the hold, probably raided from a civilian transport. It looks like their haul was worth the damage.`

Reviewer 3: The first sentence is a bit run-on, so a period would probably work better than a semicolon.
```suggestion
			`	As it comes to a stop in a maintenance bay, the spaceport's engineering shuttles appear from all over and begin buzzing around the Bastion, assessing the damage and beginning the work on repairs. Meanwhile, its crew disperse from the ship and immediately begin unloading a multitude of busted-up crates and containers from the hold, probably raided from a civilian transport. It looks like their haul was worth the damage.`
```

> 			`	Within the hour, the work shuttles scatter from the ship, having done their work replacing the battered hull and blown shield emitters, and the Bastion maneuvers out of the bay almost as haphazardly as it did when it entered.`

Reviewer 3: Comma overload here. You might want to split it up into a couple of sentences, or you could use parentheses or dashes for the centre section instead.

Author: ```suggestion
			`	Within the hour the work shuttles scatter from the ship as they finish patching up its battered hull, and the Bastion maneuvers out of the bay almost as haphazardly as it did when it entered.`
```
